app/common/MandationStatus.scala

```diff
@@ -19,5 +19,6 @@ package common
 object MandationStatus {
 
   val nonMTDfB = "Non MTDfB"
+  val mandated = "MTDfb"
```

Comment:
If you change the logic on the view (see comment) you don't need this val. (This value is actually meant to be 'MTDfB Mandated' anyway)